<?xml version="1.0"?>
<sdf version="1.4">
  <world name="default">
    <plugin name="ros_get_coord" filename="libros_get_coord.so"/>
  </world>
</sdf>
